web_design/fourth_mockup_design/html/web.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Web Page</title>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;
+        0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,800;1,900&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="../css/webstyle.css">
+    </head>
+    <body>
+        <main class="main-content">
+            <nav class="navbar">
+                <a class="food" href="/">Food<span class="bag">Bag</span></a>
+                <ul class="navbar-list">
+                    <li class="navbar-list-item"><a class="navbar-list-link" href="#">Cart</a></li>
+                    <li class="navbar-list-item"><a class="navbar-list-link" href="#">Login</a></li>
+                    <li><a class="navbar-list-btn" href="#">Sign Up</a></li>
+                </ul>
+            </nav>
+            <section>
+                <div class="type">
+                    <p>
+                        <span class="breadcrumb-item text-gray">Home</span> 
+                        <span class="breadcrumb-item text-gray">></span> 
+                        <span class="breadcrumb-item text-gray">Cart</span>
+                        <span class="breadcrumb-item text-gray">></span>
+                        <span class="breadcrumb-item text-black">Verification</span>
+                    </p>
+                </div>
+            </section>
+            <section>
+                <p class="verification">Verification</p>
+                <p class="verification-text">comfirm address & payment</p>
+                <div class="delivery-wrapper">
+                    <div>
+                        <p class="delivery"><strong>Delivery Address</strong>
+                            <p class="delivery-text">Your order will be delivered to your address</p>
+                        </p>
+                    </div>
+                    <div class="delivery-btn">
+                        <img src="../images/button.png" width="20px" height="20px" alt="button.png"/></p>
+                    </div>
+                </div>
+            </section>
+            <section class="number-container">
+                <label class="number-bar" for="phone number"><strong>Phone number</strong></label>
+                <div class="number-wrapper">
+                    <select class="number-bar-box" name="phone number" id=" phone number">
```

Comment:
The value for the `name` and `id` attributes should not have space. E.g it should be `phone_number` not `phone number`